.github/workflows/publish.yml

```diff
@@ -2,23 +2,38 @@
 name: Publish
 on:  # yamllint disable-line rule:truthy
   push:
-    branches:
-      - main
+    # branches:
+    #   - main
```

Comment:
why?

Reply:
This is just temporary to tigger the action on a PR push. I'll drop all `DEBUG` commits before this merges.